src/h3/api/_api_template.py

```diff
@@ -17,22 +17,43 @@ def _api_functions(
         _globals,
 ):
     def versions():
-        return c.versions()
+        """ Return version numbers for both the Python library
+        and the underlying H3 C library.
+
+        The two versions should match exactly.
```

Comment:
Is this correct? Our versioning strategy has been to allow the bindings to set their own patch versions - otherwise it's difficult to bump the binding for binding-specific fixes.

Reply:
Yep, I agree with @nrabinowitz the major and minor should match, but patch is freely updated on the bindings.

Reply:
Got it. I'll update.